lib/ReactViews/Analytics/InvokeFunction.jsx

```diff
@@ -32,6 +32,10 @@ const InvokeFunction = React.createClass({
         this.initializeParameters(nextProps);
     },
 
+    componentWillUpdate(nextProps, nextState) {
+        this.initializeParameters(nextProps);
+    },
+
```

Comment:
* It seems strange to knockout.track the same thing in both componentWillMount and initializeParameters

That does seem odd. I am going to remove the knockout.track in initializeParameters and what happens.

* initializeParameters also "enable[s] the context item, if any" - can that be moved somewhere more appropriate?

I'll have a look at moving this.

* I'm worried that the initializeParameters will overwrite desirable parameters with the default values in some situation we haven't come across yet.

Yeah, fair enough. I will see if I can fix this.

Thanks @RacingTadpole!